### ISSUE TITLE
Entities are constantly turning unavailable

### ISSUE THREAD

**Issue by XalaTheShepard** [skodaconnect/homeassistant-skodaconnect#165]
**Describe the bug**
After the latest update of the Skodaconnect integration and HA Core (2023.1.0) the entities of my Skoda Superb constantly become unavailable. The only solution is a reload of the integration. Might be an issue at the Skoda server, but not sure about this. 

**Latest working release**
If the problem started after updating to a new release, please state latest working release.
1.1.4 of Skoda Connect and 2022.12.9 of HA Core

**Debug logs**
If applicable, add debug logs. See [README](https://github.com/lendy007/homeassistant-skodaconnect#enable-debug-logging) on how to enable debug logs with response debugging.
No logs about the issue

**Installation:**
 - Home Assistant variant: [e.g. Hass OS, Docker, Core, Supervised]
 - Python version [e.g. 3.7, 3.8]
 - Python library versions ["python3 -m pip list"]
 HassOS in VM

**Additional context**

**Comment by flofricke** [comment]
**Describe the bug**
I have the same problem. After a while the entities become unavailable.

**Latest working release**
I switched from HA OS to HA Docker (from 2022.12.9 to 2023.1.0) and from Skoda Connect 1.1.2 to 1.2
The former was fully functional.

**Debug logs**
Logger: skodaconnect.connection
Source: custom_components/skodaconnect/__init__.py:907
Integration: Skoda Connect (documentation, issues)
First occurred: 5. Januar 2023 um 21:35:06 (1725 occurrences)
Last logged: 07:09:18

Could not refresh tokens: Failed token refresh
Could not fetch operation list, error: Failed to set token for "vwg": No valid tokens for client "vwg"

**Installation**
Docker, stable

**Comment by Farfar** [comment]
There seems to be an issue with VW-Group token refresh (the API token for all non Enyaq cars).
The issue was probably always there but weren't noticed before the token handling change.
I have pushed new library version that will hopefully fix this issue.

**Comment by Farfar** [comment]
Update to version 1.2.1 of integration and it will require the new library version to be installed (1.3.2)

**Comment by flofricke** [comment]
Thanky you, i have updated the version. I will update tomorrow if this has fixed the problem.

**Comment by MarcoGos** [comment]
@Farfar It looks like version 1.2.1 did the trick. Now 4 hours since this update and the car is still available.

**Comment by flofricke** [comment]
Same for me, works perfectly again. Thank you

**Comment by Farfar** [comment]
The "Last connected" is still not handled correctly and sometimes throws errors when invalid value is received from API. Working on a good fix for that.

**Comment by Farfar** [comment]
Duplicate of #157

**Comment by ChristophCaina** [comment]
@all that suffers from sensors that become unavailable:

The last time, when I had a similar issue, the battery state of charge was one of the affected sensors (but not only).
I've checked the Skoda app - and - of the battery information was also not visible there.
So I knew, it was (again) something related to the unstable service provided by skoda / vw.

anyway - even on the next day, the sensors in the integration remained unknown.
Again, I opened the app - but there, all information were available.
So I reloaded the integration, and just after that the information were available again in HomeAssistant.

So... it seems that there is at least some kind of issue that the integration is not able to update the entities, when there was something wrong with the servers.
Because before I opened the app, I've tried to reload the integration which did not help.
Only after opening the app (no consent to confirm, etc.) the states became available again.

1. maybe, this does help the one or the other as a work around
2. maybe, it will help to improve the reiability of the integration (althought, I cannot provide any further logs at the moment :-( )
